examples/common.cpp

```diff
@@ -92,414 +92,315 @@ void process_escapes(std::string& input) {
+                params.use_mmap = false;
+            } else if (arg == "--lora-base") {
+                validate_params(arg, argc, i, argv);
+                params.lora_base = argv[i];
+            } else if (arg == "-i" || arg == "--interactive") {
+                params.interactive = true;
+            } else if (arg == "--embedding") {
+                params.embedding = true;
+            } else if (arg == "--interactive-first") {
+                params.interactive_first = true;
+            } else if (arg == "-ins" || arg == "--instruct") {
+                params.instruct = true;
+            } else if (arg == "--multiline-input") {
+                params.multiline_input = true;
+            } else if (arg == "--color") {
+                params.use_color = true;
+            } else if (arg == "--mlock") {
+                params.use_mlock = true;
+            } else if (arg == "--gpu-layers" || arg == "-ngl" || arg == "--n-gpu-layers") {
+                validate_params(arg, argc, i, argv);
+    #ifdef LLAMA_SUPPORTS_GPU_OFFLOAD
+                params.n_gpu_layers = std::stoi(argv[i]);
+    #else
+                fprintf(stderr, "warning: not compiled with GPU offload support, --n-gpu-layers option will be ignored\n");
+                fprintf(stderr, "warning: see main README.md for information on enabling GPU BLAS support\n");
+    #endif
+            } else if (arg == "--main-gpu" || arg == "-mg") {
+                validate_params(arg, argc, i, argv);
+    #ifdef GGML_USE_CUBLAS
+                params.main_gpu = std::stoi(argv[i]);
+    #else
+          fprintf(stderr, "warning: llama.cpp was compiled without cuBLAS. It is not possible to set a main GPU.\n");
+    #endif
+            } else if (arg == "--tensor-split" || arg == "-ts") {
+                validate_params(arg, argc, i, argv);
+    #ifdef GGML_USE_CUBLAS
+                std::string arg_next = argv[i];
+
+                // split string by , and /
+                const std::regex regex{R"([,/]+)"};
+                std::sregex_token_iterator it{arg_next.begin(), arg_next.end(), regex, -1};
+                std::vector<std::string> split_arg{it, {}};
+                GGML_ASSERT(split_arg.size() <= LLAMA_MAX_DEVICES);
+
+                for (size_t i = 0; i < LLAMA_MAX_DEVICES; ++i) {
+                    if (i < split_arg.size()) {
+                        params.tensor_split[i] = std::stof(split_arg[i]);
+                    } else {
+                        params.tensor_split[i] = 0.0f;
+                    }
+                }
+    #else
+          fprintf(stderr, "warning: llama.cpp was compiled without cuBLAS. It is not possible to set a tensor split.\n");
+    #endif // GGML_USE_CUBLAS
+            } else if (arg == "--low-vram" || arg == "-lv") {
+    #ifdef GGML_USE_CUBLAS
+                params.low_vram = true;
+    #else
+          fprintf(stderr, "warning: llama.cpp was compiled without cuBLAS. It is not possible to set lower vram usage.\n");
```

Comment:
The indentation of these `fprintf` was recently fixed in master, remember to fix it here as well after rebasing.